### ISSUE TITLE
Dockerize Application:Trying to follow quickstart guide to run the application quickly, having problems

### ISSUE THREAD

**Issue by jgill-compucloud** [VivekPa/AlphaAI#26]
Tried following the quickstart guide to run the application quickly but running into many software dependency issues. Have you considered dockerizing this application? 

I have a background in this area, would be interested in contributing to the project to add that capability if you are open to incorporating this as a feature. 

Let me know.

**Comment by Patrick-Gourdet** [comment]
Yes, I agree, but you can also just create a virtual env using pip then pip install using the requiements.txt  that will install all dependencies without effecting your primary python environment.
https://www.dabapps.com/blog/introduction-to-pip-and-virtualenv-python/#:~:text=virtualenv%20is%20a%20tool%20for,time%20on%20the%20same%20machine. This link has a good step by step to initialize.
